explore/src/main/scala/explore/targeteditor/AsterismEditor.scala

```diff
@@ -108,19 +108,19 @@ object AsterismEditor extends AsterismModifier:
 
         <.div(
           ExploreStyles.AladinFullScreen.when(fullScreen.get.value),
-          if (props.readonly) EmptyVdom
-          else
```

Comment:
Why do we want to display the target selection popup if they can't do anything with it?

Reply:
It makes the layout simpler if we always know there is an element.
But yes we could hide it as well and tweak the css a bit more

Reply:
Not a big deal, I guess. Keeping CSS simpler is a good goal.

Reply:
Would an empty `span` work as well as the button for keeping the CSS simple?